.github/workflows/nodejs.yml

```diff
@@ -29,6 +29,6 @@ jobs:
       test-command: npm run coverage:ci
       post-test-steps: |
         - name: Upload coverage report to Codecov
-          uses: codecov/codecov-action@v3
-    strategy:
-      fail-fast: false
```

Comment:
This was wrong. Fail-fast is setup inside the pkgjs action and is `false` by default: https://github.com/pkgjs/action/blob/main/docs/USAGE.md#strategy-fail-fast